benchmarks/remove_all.js

```diff
@@ -14,6 +14,28 @@ var words = require('./words').words;
 var range = require('./words').range;
 var api = require('./shared');
 
+var nativeObjectRemoveAll = function(keys, order) {
+  var h = api.nativeObjectFrom(keys);
+  return function() {
+    var c;
+    for(var i = 0, len = order.length; i < len; ++i) {
+      c = Object.assign({}, h);
+      delete c[order[i]];
+    }
+  };
+};
+
+var nativeMapRemoveAll = function(keys, order) {
+  var h = api.nativeMapFrom(keys);
+  return function() {
+    var c;
+    for(var i = 0, len = order.length; i < len; ++i) {
+      c = new Map(h);
```

Comment:
These still need some work. It should be the cost to remove every element (without replacement) from a map. The current implementation starts with a new map every time. I believe it should be something like this:
```
var c =  new Map(h); // create full map at start
// now remove elements one at a time until all have been removed
for(var i = 0, len = order.length; i < len; ++i) {
    c = new Map(c); // copy the old map to make sure the original reference is not mutated.
    c.delete(order[i]);
}
```